account_block_account/models/account_block_mixin.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class AccountBlockMixin(models.AbstractModel):
+    """Inherit this from models that can be blocked"""
+    _name = 'account.block.mixin'
+    _description = 'Mixin for accounting entities that can be blocked'
+
+    blocked = fields.Boolean(default=False)
```

Comment:
It seems that `blocked` is already in use for `account.move.line` in [10.0](https://github.com/OCA/OCB/blob/10.0/addons/account/models/account_move.py#L401) and even [8.0](https://github.com/OCA/OCB/blob/8.0/addons/account/account_move_line.py#L495). Should we rename to something less likely to be hijacked, such as `write_blocked`

Reply:
Would it even be better for extensibility to make a computed field, so that people could extend it to their own implementation of "Blocked"? Perhaps some custom implementations would want it as a state, and that way they could inherit the function to store where they wish.

Reply:
i was aware of this property and named it like that exactly because it's called like this on move lines. it's the same functional claim: when it's blocked, you can't do anything with it. note that in the concept of this addon, move lines won't inherit this mixin, but the [mixin for referring to something that might be blocked](https://github.com/OCA/account-financial-tools/pull/616/files#diff-e0919135daded1487502e2a17a0a8895R8)
for the computed field: we're in v8+, no need for provisions for computed fields. the next addon overriding the field adding a compute and inverse function makes it a computed field, simple as that

Reply:
myself i'd have preferred to call it `locked` accounts, but if blocking is established, i rather go for conformity than for preciseness of language